dockerfiles/Dockerfile.cross

```diff
@@ -1,3 +1,3 @@
-FROM    dockercore/golang-cross:1.9.5@sha256:4d090b8c2e6d369a48254c882a4e653ba90caaa0b758105da772d9110394d958
+FROM    dockercore/golang-cross@sha256:7ea2b1df222604a27b736bf6a0bf65936798838b4596f8dbfaf3e7a9153aa32e
```

Comment:
If someone can tag this as `1.10.2`, then I'll update this reference 👍

Reply:
Done in git, hub auto build is in progress :wink: